doc/Development_Documentation/21_Deployment/03_Configuration_Environments.md

```diff
@@ -35,7 +35,8 @@ The following options are available:
 
 Available environment variables are: 
 ```bash
-PIMCORE_WRITE_TARGET_IMAGE_THUMBNAILS=settings-store
 PIMCORE_WRITE_TARGET_CUSTOM_REPORTS=settings-store
+PIMCORE_WRITE_TARGET_DOCUMENT_TYPES=settings-store
```

Comment:
Can't add a suggestion, but this should be added in the list above (line 27), see: #10217